plugins/e57/io/E57Writer.cpp

```diff
@@ -162,6 +162,11 @@ void E57Writer::write(const PointViewPtr view)
 
 bool E57Writer::processOne(PointRef& point)
 {
+    // Update bounding box
```

Comment:
Alright, this is because for preview to actually function, it needs metadata (or read the whole file). This is here to add the missing info at write time so that the plugin is complete in a way. Will ammend the title.